docs/whats-new/release-candidate-1.md

```diff
@@ -0,0 +1,834 @@
+---
```

Comment:
Does this file need a different name? `release-candidate-1` seems pretty generic. Why not have `9` in the file name?